swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md

```diff
@@ -46,17 +46,17 @@ The `in-memory` data store supports the following Ballerina types.
 
 The In-Memory data store does not require any configuration. 
 
-## MySQL
+## Relational Databases(MySQL, MSSQL)
 
-The MySQL data store is a relational database management system that stores data in tables. It is useful for storing data in a relational format. It is not the default data store for `bal persist`. Therefore, you need to explicitly specify the data store when initializing `bal persist` in your application as follows.
+The data store is a relational database management system that stores data in tables. It is useful for storing data in a relational format. It is not the default data store for `bal persist`. Therefore, you need to explicitly specify the data store when initializing `bal persist` in your application as follows.
 
 ```shell
-$ bal persist init --datastore mysql
+$ bal persist init --datastore [mysql/mssql]
 ```
 
 ### Supported Ballerina types
 
-The following table lists the Ballerina types supported by the MySQL data store and the corresponding SQL types used to store the data in the database.
+The following table lists the Ballerina types supported by the data store and the corresponding SQL types used to store the data in the database.
 
 |  Ballerina type  |    SQL type     |
```

Comment:
There is a slight variation in the SQL types used for MSSQL vs MySQL. Should we include those?

Reply:
What is the variation, we better include them

Reply:
`float` --> `FLOAT`
`boolean` --> `BIT`
`decimal` --> `DECIMAL(38, 30)`
`byte[]` --> `VARBINARY(MAX)`
`time:Civil` --> `DATETIME2`
`time:Utc` --> `DATETIME2`
`enum` --> `VARCHAR` with checks.

Example for enum:
```ballerina
enum Gender {
    MALE,
    FEMALE
}
```

```sql
gender VARCHAR(6) CHECK (gender IN ('MALE', 'FEMALE')) NOT NULL,
```

Reply:
Fixed in [fe1e7d7](https://github.com/ballerina-platform/ballerina-dev-website/pull/7147/commits/fe1e7d704578895e682f108bbca67e3f3c007fe4)